raptorWeb/raptorbot/discordbot/util/raptorbot_util.py

```diff
@@ -0,0 +1,179 @@
+import discord
+from json import dump, dumps, load
+from json.decoder import JSONDecodeError
+from time import time
+import logging
+
+from asgiref.sync import sync_to_async
+from threading import Thread
+
+from raptorWeb import settings
+if settings.SCRAPE_ANNOUNCEMENT:
+    from gameservers.models import Server
+    from raptorbot.models import Announcement, ServerAnnouncement
+from raptorbot.models import DiscordGuild
+
+logging.basicConfig(filename="error.log", level=logging.DEBUG)
+
+from raptorbot.discordbot.util import raptorbot_settings
+
+async def get_server_roles(bot_instance):
```

Comment:
This function could be phased out in favor of setting server roles in a Server's model as a Field.